src/test/resources/eager/commits-variables-from-do-tag-when-partially-resolved.expected.expected.jinja

```diff
@@ -0,0 +1,2 @@
+L1: ['a']
```

Comment:
for short files like these, I generally prefer to inline them in the test so it's easier to see the input and output without flipping between 3 files.

Reply:
I don't see how defining these strings statically in the class is any different than using external files, but you seem to feel strongly about this.